test/specs/config/configuration.js

```diff
@@ -288,42 +295,49 @@ describe('config/configuration', function() {
             configuration.load({excludeFiles: ['1.js', 'app/1.js']});
             expect(!!configuration.isFileExcluded('1.js')).to.equal(true);
             expect(!!configuration.isFileExcluded('app/1.js')).to.equal(true);
-            expect(!configuration.isFileExcluded('share/1.js')).to.equal(true);
+            expect(!!configuration.isFileExcluded('share/1.js')).to.equal(true);
```

Comment:
Guess, better to add slash to `excludeFiles` items: `['/1.js', '/app/1.js']` at least before next major.